sklearnex/dispatcher.py

```diff
@@ -48,10 +48,20 @@ def get_patch_map():
         import sklearn.neighbors as neighbors_module
         import sklearn.svm as svm_module
 
+        if sklearn_check_version("1.2.1"):
```

Comment:
should this be 1.2.1 specifically or just from 1.2 ?

Reply:
yes, it should. they moved this code from fixes module to parallel module and slightly changed it exactly in 1.2.1